### PR TITLE
Dtspb 3926 data migration sols cases

### PR DESCRIPTION
**Pull request by IswaryaPepakayala**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPB-3926

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [X] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java [hunk 1]
```diff
@@ -49,29 +49,25 @@ public CaseDetails update(String authorisation, String eventId,
...
-                userDetails.getId(),
-                updatedCaseDetails.getJurisdiction(),
-                caseType,
-                caseId,
-                true,
-                caseDataContent);
-        } else {
-            return null;
-        }
+        CaseDataContent caseDataContent = CaseDataContent.builder()
```
Comment (MarcusHMCTS) [inline src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java:52]: should check the applicantOrganisationPolicy & supplementary_data is null before update

Comment (IswaryaPepakayala) [reply src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java]: done

**Review by MarcusHMCTS — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/migration/service/OrganisationsRetrievalService.java
[the remark this answers is not in the document]
Comment (IswaryaPepakayala) [reply src/main/java/uk/gov/hmcts/reform/migration/service/OrganisationsRetrievalService.java:1]: Removed

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -34,5 +35,10 @@ migration.dryrun=${MIGRATION_DRYRUN:true}
 migration.rollback.start.datetime=${MIGRATION_ROLLBACK_START_DATETIME:}
 migration.rollback.end.datetime=${MIGRATION_ROLLBACK_END_DATETIME:}
 
+prd.organisations.url=${PRD_API_URL:http://localhost:8991}
+prd.organisations.api=/refdata/external/v1/organisations
```
[the remark this answers is not in the document]
Comment (IswaryaPepakayala) [reply src/main/resources/application.properties]: Removed

**Pull request merged**